sdk/keyvault/azure-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/SecretRequestAttributes.java

```diff
@@ -18,16 +18,16 @@
     /**
      * Creates an instance of SecretRequestAttributes. Reads secretBase.notBefore, secretBase.expires and
```

Comment:
Need to update this line to say secretProperties instead of secretBase

Reply:
internal class.
Updated.